content/en/monitors/guide/non_static_thresholds.md

```diff
@@ -0,0 +1,236 @@
+
+### Monitor
+
+To count the number of indexed logs in Datadog use the following metric: [`datadog.estimated_usage.logs.ingested_events`][4].
+It is an out-of-the-box metric that can be found in your account after logs are sent to Datadog. For more information on log usage metrics, see the [Log Configuration][5] documentation.
+
+The monitor evaluates the sum over the last 5 minutes compared to the average of the last hour. It is composed of 2 queries:
+- Query A represents the last 5 minutes of the metric you want to be aware of, because of the parameter "Evaluate the … of the query over the last 5 minutes".
+- Query B represents the average value of the same metric over the past hour thanks to the moving rollup, which takes the datapoints of the last 4 hours (to have a large amount of datapoints) and calculates the average.
+
+When the value of Query A is much higher than the value of Query B, it's a sign of a substantial and sudden increase in usage which you can capture with a ratio between A and B.
+
+{{< tabs >}}
+{{% tab "UI Configuration" %}}
+
+{{< img src="monitors/guide/non_static_thresholds/spike_threshold.png" alt="Spike threshold config with two log usage metrics" style="width:100%;" >}}
+
+In the formula, divide **Query A** by **Query B** and multiply the quotient by the moving rollup to account for the difference in evaluation windows between the two queries. In this example, since the moving rollup of Query B is 4 hours (240 minutes), you multiply the formula with 48 (240 minutes / 5 minute evaluation window).	
+
+The threshold here is 2, to get an alert when the difference in value is double or higher.
+
+{{% /tab %}}
+
+{{% tab "JSON Example" %}}
+```
+{
+	"name": "[Spike threshold] Log Spike",
+	"type": "query alert",
+	"query": "sum(last_5m):(sum:datadog.estimated_usage.logs.ingested_events{env:prod}.as_count() / moving_rollup(sum:datadog.estimated_usage.logs.ingested_events{env:prod}.as_count(), 14400, 'sum')) * 48 > 2",
+	"message": "Be careful! There has been a difference of {{value}} in Log ingestion between now and the past 4 hours!",
+	"tags": [],
+	"options": {
+		"thresholds": {
+			"critical": 2
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": false
+	}
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+## Reference threshold
+
+### Context
+You are the QA team lead, in charge of the checkout process of your e-commerce website. You want to make sure that your customers can have a good experience and can purchase your products without any issues. One indicator of that is the error rate.
+
+The traffic is not the same throughout the day, so 50 errors/minutes on a Friday evening is not as worrying as 50 errors/minute on a Sunday morning. Monitoring an error rate rather than the errors themselves gives you a reliable view of what healthy and unhealthy metrics look like.
```

Comment:
```suggestion
The traffic is not the same throughout the day, so 50 errors/minute on a Friday evening is less worrying than 50 errors/minute on a Sunday morning. Monitoring an error rate rather than the errors themselves gives you a reliable view of what healthy and unhealthy metrics look like.
```